go.sum

```diff
@@ -1586,8 +1586,8 @@ github.com/smartcontractkit/libocr v0.0.0-20220812191430-db92a9fdaa52 h1:Nac1UCK
 github.com/smartcontractkit/libocr v0.0.0-20220812191430-db92a9fdaa52/go.mod h1:5JnCHuYgmIP9ZyXzgAfI5Iwu0WxBtBKp+ApeT5o1Cjw=
 github.com/smartcontractkit/ocr2keepers v0.4.8 h1:pvCDWIsYaSJI/M6qw0xIIXpHs5MPQaYEpmWsJiyi6LM=
 github.com/smartcontractkit/ocr2keepers v0.4.8/go.mod h1:6wSlAWYBmlJM7bDZ4Ewc5J5AJQszIRGDNaUAw35r3qc=
-github.com/smartcontractkit/ocr2vrf v0.0.0-20221103213748-db2f67990850 h1:cZX0weC3QmHlS+stmLaliNgd4GfemmEl3tziZSYXKk8=
-github.com/smartcontractkit/ocr2vrf v0.0.0-20221103213748-db2f67990850/go.mod h1:NKkp8yf3trq+hJe/gn2Q1dd4abZvcYavUKPzYJCgVew=
+github.com/smartcontractkit/ocr2vrf v0.0.0-20221108202008-9139c7142460 h1:8oERPLVYE/F8rtBoo8IRqgx7MjuvNh2TNShq8dL0PF8=
+github.com/smartcontractkit/ocr2vrf v0.0.0-20221108202008-9139c7142460/go.mod h1:NKkp8yf3trq+hJe/gn2Q1dd4abZvcYavUKPzYJCgVew=
```

Comment:
LGTM - probably need to update this after merging ocr2vrf

Reply:
yup yup